src/include/mpir_group.h

```diff
@@ -60,8 +60,6 @@ struct MPIR_Group {
     MPII_Group_pmap_t *lrank_to_lpid;   /* Array mapping a local rank to local
                                          * process number */
     int is_local_dense_monotonic;       /* see NOTE-G1 */
-    const char *pset_name;      /* set to one of the pre-configured pset names,
-                                 * or NULL if it is not directly from a pset */
```

Comment:
I guess this makes sense for world and self. If we ever support different psets, I suppose we can just add it back.

Reply:
If we ever support different psets, more work or redesign will be required. I am not sure there is any practical benefit with other pets. For example, if an application wants to have separate sessions over disjoint process sets, it is probably easier to launch them separately and have them join via `MPI_Open_port/MPI_Comm_connect/MPI_Comm_accept`. Or one could simply launch separate worlds with dynamic processes. The pre-set topology can be easily managed by process manager. So my opinion is -- until there is a convincing use case, let's assume we'll never support additional psets. But if we ever will support that, we should re-design then.